### PR TITLE
docs: fix .md links

### PR DESCRIPTION
**Pull request by mpoke**
## Description

Closes: NA

---

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [x] included the correct `docs:` prefix in the PR title
- [x] targeted the correct branch (see [PR Targeting](https://github.com/cosmos/gaia/blob/main/CONTRIBUTING.md#pr-targeting))
- [ ] provided a link to the relevant issue or specification
- [x] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items.*

I have...

- [ ] Confirmed the correct `docs:` prefix in the PR title
- [ ] Confirmed all author checklist items have been addressed 
- [ ] Confirmed that this PR only changes documentation
- [ ] Reviewed content for consistency
- [ ] Reviewed content for thoroughness
- [ ] Reviewed content for spelling and grammar
- [ ] Tested instructions (if applicable)

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -304,10 +304,6 @@ proto-gen:
 	@echo "Generating Protobuf files"
 	@sh ./proto/scripts/protocgen.sh
 
-proto-doc:
```
Comment (mpoke) [inline Makefile:307]: Removed as it was not working and could not update `docs/proto/proto-docs.md`

File: docs/hub-tutorials/upgrade-node.md
Comment (mpoke) [inline docs/hub-tutorials/upgrade-node.md:1]: Is the information in this file still accurate? Is it consistent with the info in `docs/migration`? cc @mmulji-ic

**Review by yaruwangway — APPROVED** [review]

File: .github/workflows/md-link-checker.yml [hunk 2]
```diff
@@ -1,5 +1,6 @@
 name: Check Markdown links
 on: 
+  workflow_dispatch:
```
Comment (MSalopek) [inline .github/workflows/md-link-checker.yml:3]: This just allows you to manually run the check action. Why remove?

**Review by MSalopek — APPROVED** [review]

File: .github/workflows/md-link-checker.yml (hunk 2 above)
Comment (mmulji-ic) [reply .github/workflows/md-link-checker.yml:3]: It was added?

File: docs/hub-tutorials/upgrade-node.md
Comment (mmulji-ic) [reply docs/hub-tutorials/upgrade-node.md:1]: No it needs a revision and some consolidation.

File: docs/migration/README.md [hunk 3]
```diff
@@ -12,6 +12,6 @@ markdown-link-check-disable
 
 This directory houses Cosmos Hub major upgrade migration instructions.
 
-- [Upgrading from `cosmoshub-2` to `cosmoshub-3`](cosmoshub-2.md)
-- [Upgrading from `cosmoshub-3` to `cosmoshub-4`](cosmoshub-3.md)
+- [Upgrading from `cosmoshub-2` to `cosmoshub-3`](./cosmoshub-2/cosmoshub-2.md)
```
Comment (mmulji-ic) [inline docs/migration/README.md:15]: @mpoke thxs for fixing this

File: .github/workflows/md-link-checker.yml (hunk 2 above)
Comment (MSalopek) [reply .github/workflows/md-link-checker.yml:3]: I guess I developed daltonism this morning :D

Thanks for adding it!

**Pull request merged**